### PR TITLE
Initial File Structure reorg

### PR DESCRIPTION
**Pull request by henkvancann**

### CODE REVIEW

File: Docs/Lightning.md [hunk 1]
```diff
@@ -19,64 +19,15 @@ First you need to install c-lightning, to do that follow [this](https://github.c
...
+Pick the guide below for the device your're running your node on:
+ - [Mac Osx](./Tor.md/#Connecting-over-MacOS)
+ - [Windows 10](./Tor.md/#Connecting-over-Tor-Windows-10)
+ - [Debian 10](./Tor.md/#Connecting-over-Tor-Linux-Debian-10)
 
-You will need Tor running on the node. On linux `sudo apt install tor` works, on macOS `brew install tor` does the trick.
+Find the suggested Authentication settings on the device **running FN** [here](./Authentication.md/#On-the-device-running-FN).
+Find the suggested `bitcoin.conf` settings for FN [here](./Howto.md/#Bitcoin-Core-settings). For *Lightning* see below, or click [here](#Connect-Fully-Noded).
 
-Boot tor as a service:
```
Comment (Fonta1n3) [inline Docs/Lightning.md:26]: why delete this? these are specific to setting up lightning as a hidden service

File: Docs/Tests.md [hunk 2]
```diff
@@ -0,0 +1,157 @@
+# Tests
+
+## Core Test group
+
+Spring 2020 we installed a Core test group in Signal for FN2 (now called 'Gordian Wallet'). See [Contributing](../Readme.md#Contributing).
```
Comment (Fonta1n3) [inline Docs/Tests.md:5]: Gordian Wallet tests are quite different to FN tests, they prob should not be included

File: Docs/Tor.md [hunk 3]
```diff
@@ -0,0 +1,219 @@
+## Connecting over Tor 
+ - [macOS](#Connecting-over-Tor-macOS)
+ - [Windows 10](#Connecting-over-Tor-Windows-10)
+ - [Debian 10](#Connecting-over-Tor-Linux-Debian-10)
+ 
+#### Optional Tor v3
```
Comment (Fonta1n3) [inline Docs/Tor.md:6]: tor v3 is not optional

File: Docs/Tor.md [hunk 4]
```diff
@@ -0,0 +1,219 @@
...
+
+HiddenServiceDir /usr/local/var/lib/tor/fullynoded/test
+HiddenServiceVersion 3
+HiddenServicePort 18332 127.0.0.1:18332
+
+HiddenServiceDir /usr/local/var/lib/tor/fullynoded/regtest
+HiddenServiceVersion 3
+HiddenServicePort 18443 127.0.0.1:18443
+
+HiddenServiceDir /usr/local/var/lib/tor/fullynoded/lightning
```
Comment (Fonta1n3) [inline Docs/Tor.md:46]: can probably keep this one in lightning.md to keep confusion to a minimum, lightning setup is a totally different animal

File: Docs/Tor.md [hunk 5]
```diff
@@ -0,0 +1,219 @@
...
+
+HiddenServiceDir "C:/Tor/fullynoded/test/"
+HiddenServiceVersion 3
+HiddenServicePort 18332 127.0.0.1:18332
+
+HiddenServiceDir "C:/Tor/fullynoded/regtest/"
+HiddenServiceVersion 3
+HiddenServicePort 18443 127.0.0.1:18443
+
+HiddenServiceDir "C:/Tor/fullynoded/lightning/"
```
Comment (Fonta1n3) [inline Docs/Tor.md:107]: same as above, connecting lightning and bitcoin core are different animals

File: Docs/Lightning.md (hunk 1 above)
Comment (henkvancann) [reply Docs/Lightning.md:26]: I moved them to the connect-node.md, for the top of my head. I do not delete anything that has purpose, at least not intentionally

**Review by Fonta1n3 — APPROVED** [review]

**Pull request merged**